scrapers/ExCoGi.yml

```diff
@@ -0,0 +1,32 @@
+name: "ExCoGi"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - exploitedcollegegirls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class='info']
+    scene:
+      Title: $info/h4/strong/text()
+      Details: //article[@id='about']/p/text()
+      Image:
+        selector: //img[@class='img-responsive center-block main-preview']/@src
+        postProcess:
+          - replace:
+            - regex: ^
+              with: https://exploitedcollegegirls.com
+      Studio:
+        Name: Exploited College Girls
```

Comment:
forgot the `fixed`
``yaml
      Studio:
        Name: 
          fixed: Exploited College Girl
```